src/clj/rems/css/styles.clj

```diff
@@ -96,14 +100,16 @@
                            :background-size :contain
                            :background-repeat :no-repeat
                            :background-position [[:center :center]]}]
-                         [:.logo
-                          {:height (u/px 150)}]))
+                         [:.logo {:height logo-height
+                                  :min-height logo-height}]))
    (stylesheet/at-media {:max-width (u/px 870)}
                         [:.user-widget [:.icon-description {:display "none"}]])
    (stylesheet/at-media {:min-width (u/px 480)}
                         [:.commands {:white-space "nowrap"}])
    (stylesheet/at-media {:prefers-reduced-motion :reduce}
-                        [:body {:scroll-behavior :auto}])))
+                        [:body {:scroll-behavior :auto}])
+   (stylesheet/at-media {:max-width (u/px 480)}
+                        [:.navbar-brand {:display "none"}])))
```

Comment:
This rule could be placed along with the `.commands` just above. They share the breakpoint (and should share the same).